src/styles/styles/header.scss

```
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  justify-content: space-between;
+
+  &__logo-link {
+    line-height: 0;
+  }
+
+  &__list{
+    display: flex;
+    box-sizing: border-box;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+   display: inline-block;
+   height: 60px;
+   font-weight: 500;
+   font-size: 12px;
+   line-height: 60px;
+   text-align: center;
+   color: $main-color;
+   text-transform: uppercase;
+   transition: 0.3s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transition-time;
```